### DISCUSSION TITLE
window 10 pro

### DISCUSSION THREAD

**Discussion by Helsonoh** [Chia-Network/chia-blockchain#7392] [Support]
hi, i using window 10 pro,
when i syncing chia gui  my pc will auto restart...
this morning syncing chia gui must auto restart 2 or 3 time
what the problem?

my pc spec :
amd 5950
ram : 64gb
asus b550 plus
window 10 pro

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
